src/main/scala/bank/Controller.scala

```diff
@@ -27,6 +27,18 @@ class Controller extends DefaultJsonProtocol with SprayJsonSupport {
     }
   }
 
-  def allRoutes: Route = createAccount
+  def getAccountByAccountNum: Route = get {
+    path ("api" / "accounts" / Segment) { accountNumber=>
+      accounts.find(_.accountNumber == accountNumber) match {
+        case Some(accountFound)=>
+          complete(OK, s"Account found: ${accountFound.toJson}".toJson )
```

Comment:
you are mixing plain string with json, what will happen is the response will be written like below:
`{\"accountNumber\":\"1234\" ....`
instead of
`{"accountNumber": "1234" ...`

Reply:
resolved